src/client/src/Services/ApiClient.ts

```diff
@@ -13,15 +15,21 @@ export const useAuthenticatedClient = () => {
           if(response.status === 401) {
               signOut();
           }
+          if(response.status === 403) {
+            router.push("/unauthorised");
```

Comment:
Rather than the apiClient being aware of the routing and mechanism the app is using for  unauthed requests it might be worth following the pattern of providing an unauthorised() callback that would have the advantage of separating the conerns and giving a single place to update if we change the route or mechanism for handling unauthorised requests